src/config/index.tsx

```diff
@@ -1,11 +1,9 @@
 import Constants from "expo-constants";
 
-export const SENTRY_DSN: string = Constants.manifest?.extra?.sentryDsn;
-export const IS_STORYBOOK_VIEW =
-  Constants.manifest?.extra?.storybook ||
-  (Constants.manifest?.releaseChannel || "").indexOf("storybook") > -1;
-export const IS_MOCK = Constants.manifest?.extra?.mock;
+export const SENTRY_DSN: string = Constants.manifest2?.extra?.expoClient?.extra?.eas?.sentryDsn;
```

Comment:
https://docs.expo.dev/eas-update/environment-variables/#setting-and-getting-environment-variables-when-building sdk v45 has a very very stupid path for env vars